pkg/glusterfs/identityserver.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/golang/glog"
 )
 
+//IdentityServer is the struct with the reference to glusterfs CSI driver.
 type IdentityServer struct {
```

Comment:
>//NodeServer of Glusterfs CSI driver with supported methods of CSI node server spec.

@humblec can you add the same comment here also as you added for `NodeServer` struct.

Reply:
@madhu-1 made it consistent

Reply:
@humblec I prefer your old comment. it looks more meaningful as per implementation, please let me know your thoughts on this